src/main/java/com/groupseven/sysc4806project/BookstoreController.java

```diff
@@ -39,4 +42,63 @@ public String home(
         model.addAttribute("books", books);
         return "index";
     }
+
+    @PostMapping("/add-book")
+    public String add(
+            @RequestParam String isbn,
+            @RequestParam String title,
+            @RequestParam(defaultValue = "") String description,
+            @RequestParam String author,
+            @RequestParam String publisher,
+            @RequestParam(defaultValue = "0") Integer inventory,
+            @RequestParam(required = false) MultipartFile image
+    ) throws Exception {
+        if (inventory < 0) {
+            inventory = 0;
+        }
+        if(isbn == null) {
```

Comment:
These checks will never be true as Spring will throw an exception if these params are missing before we get here

Reply:
Then I don't think we need it anyway, since spring will check for the missing parameters

Reply:
I agree. If the checks are not required we should take them out

Reply:
Alright, fair enough